### PR TITLE
add way to make static custom reports of static data sources

### PR DESCRIPTION
**Pull request by czue**
and setup the new MVP reports. probably easiest read commit by commit.

@NoahCarnahan probably best to review, and in case you want to use this for abt.

cc @katembu @emord

### CODE REVIEW

**Comment by czue** [comment]
(don't merge yet - just realized i didn't test/implement part of this)

**Comment by czue** [comment]
ok should be good now

File: settings.py [hunk 1]
```diff
@@ -1351,11 +1356,11 @@
 
 CUSTOM_UCR_EXPRESSIONS = [
     ('abt_supervisor', 'custom.abt.reports.expressions.abt_supervisor_expression'),
-    ('mvp_medical_cause', 'custom.mvp_ucr.reports.expressions.medical_cause_expression'),
-    ('mvp_no_treatment_reason', 'custom.mvp_ucr.reports.expressions.no_treatment_reason_expression'),
-    ('mvp_treatment_provider_name', 'custom.mvp_ucr.reports.expressions.treatment_provider_name_expression'),
-    ('mvp_treatment_place_name', 'custom.mvp_ucr.reports.expressions.treatment_place_name_expression'),
-    ('mvp_death_place', 'custom.mvp_ucr.reports.expressions.death_place_expression'),
+    ('mvp_medical_cause', 'mvp.ucr.reports.expressions.medical_cause_expression'),
```
Comment (emord) [inline settings.py]: Why don't these need `custom._legacy` ?

Comment (czue) [inline settings.py]: that directory gets automatically added to the path (https://github.com/dimagi/commcare-hq/blob/master/manage.py#L43)

**Comment by emord** [comment]
Looks good to me will let @NoahCarnahan merge

**Comment by czue** [comment]
@NoahCarnahan is offline today and would be nice to get this in. how do you feel about merging and i'll fix anything he identifies after the fact? don't think there's any way this will break anything else

**Comment by emord** [comment]
Was just going off that you said he'd be best to review

**Comment by NoahCarnahan** [comment]
Sorry about the bad timing. Reviewing now.

**Comment by NoahCarnahan** [comment]
:+1: Looks good

**Pull request merged**